website/pages/index.js

```diff
@@ -0,0 +1,110 @@
+import Image from 'next/image';
+import Layout from 'components/Layout';
+import { getNavbarContent } from 'lib/mdx';
+import { Typography } from '@strapi/design-system/Typography';
+import { Box } from '@strapi/design-system/Box';
+import { Grid, GridItem } from '@strapi/design-system/Grid';
+import { Icon } from '@strapi/design-system/Icon';
+import ArrowRight from '@strapi/icons/ArrowRight';
+import StyledLink from 'components/StyledLink';
+import PropTypes from 'prop-types';
+import { useTheme } from '@strapi/design-system/ThemeProvider';
+
+export default function Home({ navbarContent }) {
+  const theme = useTheme();
+  return (
+    <Layout meta={{ title: 'Welcome' }} navigationContent={navbarContent ? navbarContent : {}}>
+      <Box padding={8} background="neutral0" shadow="filterShadow" hasRadius={true}>
+        <Box paddingBottom={6}>
+          <Typography variant="alpha" textColor="neutral800">
+            Welcome on Strapi Design System!{' '}
+            <span role="img" aria-label="waving hand">
+              👋
+            </span>
+          </Typography>
+        </Box>
+        <Typography variant="omega" textColor="neutral700">
+          Design System! provides components and tools to help product teams work more efficiently, and to make Strapi’s
+          applications more cohesive.
+        </Typography>
+
+        <Grid gap={5} style={{ paddingTop: 40, paddingBottom: 40 }}>
```

Comment:
Grid extends Box, you can pass `paddingTop={XX} paddingBottom={YY}` without using the DOM styles